share/BehaviorModels/dental_instrument.rigid

```diff
@@ -1,5 +1,5 @@
 Xsp 3.0
-mass 0.0106256350
+vertexMass 0.0106256350
```

Comment:
Are you sure about this change from mass to vertexMass ?

Reply:
If I let the mass, I obtain the following Warning:
```
[WARNING] [UniformMass(mass)] error reading file '/home/arnaud/dev/sofa/build/install/share/sofa/BehaviorModels/dental_instrument.rigid'. 
Unable to decode an unknow command 'mass'.
```
When changing for vertexMass, the warning is disappearing and the scene runs correctly but I don't know if the physical soundness is still the same as before.